metrics/cloudwatch2/cloudwatch2.go

```diff
@@ -0,0 +1,239 @@
+// Package cloudwatch2 emits all data as a StatisticsSet (rather than
+// a singular Value) to CloudWatch via the aws-sdk-go-v2 SDK.
+package cloudwatch2
+
+import (
+	"math"
+	"sync"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/cloudwatchiface"
```

Comment:
package cloudwatchiface 🤦‍♂️

Reply:
Yah. It's supposed to be in support of mocking out their service... But it feels so icky.